script/import_db.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+cd `dirname $0`/..
+
+DATABASE=READWRITE
+DIRECTORY=./dumps
+
+source ./admin/functions.sh
+
+if ! script/database_exists $DATABASE; then
+    count=`ls -l $DIRECTORY/mbdump*.tar.bz2 2>/dev/null | wc -l`
+    if [ $count -ne 0 ]; then
```

Comment:
You could reverse this check so that you download the files if they don't exist, then you only need to have the `InitDb.pl` line once